gtest-parallel

```diff
@@ -175,11 +236,12 @@ def run_job((command, job_id, test)):
     logger.log(str(job_id) + '> ' + line.rstrip())
 
   code = sub.wait()
-  runtime_ms = int(1000 * (time.time() - begin))
-  logger.log(str(job_id) + ': EXIT ' + str(code) + ' ' + str(runtime_ms))
+  runtime = time.time() - begin
```

Comment:
runtime_ms so we can be consistent, any sub-ms precision doesn't matter either way